bears/java/CheckstyleBear.py

```diff
@@ -43,8 +48,10 @@ def setup_dependencies(self):
             "checkstyle.jar")
 
     def create_arguments(
-            self, filename, file, config_file,
-            checkstyle_configs: known_checkstyle_or_path="google"):
+        self, filename, file, config_file,
+        checkstyle_configs: known_checkstyle_or_path="google",
+        use_spaces: bool = True, indent_size: int = 2
+    ):
```

Comment:
please no spaces around `=` for default arguments + align the `):` onto the previous line :)